mono/metadata/w32file-win32.c

```diff
@@ -515,6 +515,8 @@ mono_w32file_get_console_error (void)
 }
 #endif // HAVE_API_SUPPORT_WIN32_GET_STD_HANDLE
 
+#ifdef HAVE_API_SUPPORT_WIN32_GET_FILE_SIZE
```

Comment:
I believe this should be `if` not `ifdef`. The design was you can define 0 or 1.

Reply:
Also can just use GetFileSizeEx directly. I.e. drop Win9x/NT4 compat. GetFileSizeEx was introduced in Win2000.